include/grpcpp/security/auth_metadata_processor.h

```diff
@@ -27,29 +27,39 @@
 
 namespace grpc {
 
-/// Interface allowing custom server-side authorization based on credentials
-/// encoded in metadata.  Objects of this type can be passed to
-/// \a ServerCredentials::SetAuthMetadataProcessor().
+// Interface allowing custom server-side authorization based on credentials
+// encoded in metadata.  Objects of this type can be passed to
+// \a ServerCredentials::SetAuthMetadataProcessor().
+// Please also check out grpc::experimental::Interceptor for another way to do
```

Comment:
Please add `\a` in front of `grpc::experimental::Interceptor`, so that doxygen formats it correctly.